[2018-09-20 11:44:39,584] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:832)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:201)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:66)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1222)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1150)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.util.program.BLangFunctions.invokeNativeCallable(BLangFunctions.java:389)
	at org.ballerinalang.util.program.BLangFunctions.invokeCallable(BLangFunctions.java:260)
	at org.ballerinalang.bre.bvm.CPU.tryExec(CPU.java:407)
	at org.ballerinalang.bre.bvm.CPU.exec(CPU.java:149)
	at org.ballerinalang.bre.bvm.BLangScheduler.executeNow(BLangScheduler.java:81)
	at org.ballerinalang.util.program.BLangFunctions.invokeNonNativeCallable(BLangFunctions.java:320)
	at org.ballerinalang.util.program.BLangFunctions.invokeCallable(BLangFunctions.java:267)
	at org.ballerinalang.util.program.BLangFunctions.invokeCallable(BLangFunctions.java:247)
	at org.ballerinalang.util.program.BLangFunctions.invokePackageInitFunction(BLangFunctions.java:467)
	at org.ballerinalang.util.program.BLangFunctions.invokePackageInitFunction(BLangFunctions.java:476)
	at org.ballerinalang.util.program.BLangFunctions.invokePackageInitFunctions(BLangFunctions.java:126)
	at org.ballerinalang.BLangProgramRunner.runService(BLangProgramRunner.java:53)
	at org.ballerinalang.launcher.LauncherUtils.runServices(LauncherUtils.java:147)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:117)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:256)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:66)
 
[2018-09-20 11:47:09,668] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:832)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:201)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:66)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1222)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1150)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.util.program.BLangFunctions.invokeNativeCallable(BLangFunctions.java:389)
	at org.ballerinalang.util.program.BLangFunctions.invokeCallable(BLangFunctions.java:260)
	at org.ballerinalang.bre.bvm.CPU.tryExec(CPU.java:407)
	at org.ballerinalang.bre.bvm.CPU.exec(CPU.java:149)
	at org.ballerinalang.bre.bvm.BLangScheduler.executeNow(BLangScheduler.java:81)
	at org.ballerinalang.util.program.BLangFunctions.invokeNonNativeCallable(BLangFunctions.java:320)
	at org.ballerinalang.util.program.BLangFunctions.invokeCallable(BLangFunctions.java:267)
	at org.ballerinalang.util.program.BLangFunctions.invokeCallable(BLangFunctions.java:247)
	at org.ballerinalang.util.program.BLangFunctions.invokePackageInitFunction(BLangFunctions.java:467)
	at org.ballerinalang.util.program.BLangFunctions.invokePackageInitFunction(BLangFunctions.java:476)
	at org.ballerinalang.util.program.BLangFunctions.invokePackageInitFunctions(BLangFunctions.java:126)
	at org.ballerinalang.BLangProgramRunner.runService(BLangProgramRunner.java:53)
	at org.ballerinalang.launcher.LauncherUtils.runServices(LauncherUtils.java:147)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:117)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:256)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:66)
 
[2018-09-20 11:50:10,400] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:832)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:201)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:66)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1222)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1150)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.util.program.BLangFunctions.invokeNativeCallable(BLangFunctions.java:389)
	at org.ballerinalang.util.program.BLangFunctions.invokeCallable(BLangFunctions.java:260)
	at org.ballerinalang.bre.bvm.CPU.tryExec(CPU.java:407)
	at org.ballerinalang.bre.bvm.CPU.exec(CPU.java:149)
	at org.ballerinalang.bre.bvm.BLangScheduler.executeNow(BLangScheduler.java:81)
	at org.ballerinalang.util.program.BLangFunctions.invokeNonNativeCallable(BLangFunctions.java:320)
	at org.ballerinalang.util.program.BLangFunctions.invokeCallable(BLangFunctions.java:267)
	at org.ballerinalang.util.program.BLangFunctions.invokeCallable(BLangFunctions.java:247)
	at org.ballerinalang.util.program.BLangFunctions.invokePackageInitFunction(BLangFunctions.java:467)
	at org.ballerinalang.util.program.BLangFunctions.invokePackageInitFunction(BLangFunctions.java:476)
	at org.ballerinalang.util.program.BLangFunctions.invokePackageInitFunctions(BLangFunctions.java:126)
	at org.ballerinalang.BLangProgramRunner.runService(BLangProgramRunner.java:53)
	at org.ballerinalang.launcher.LauncherUtils.runServices(LauncherUtils.java:147)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:117)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:256)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:66)
 
[2018-09-21 01:57:39,826] ERROR {org.ballerinalang.net.http.BallerinaHTTPConnectorListener} - Error in HTTP server connector: Remote client closed the connection before initiating outbound response  
